### DISCUSSION TITLE
Updating materialized assets

### DISCUSSION THREAD

**Discussion by JSteinbauer** [dagster-io/dagster#11992] [Q&A]
Dear Dagster Community,
I wonder if there is a way to update assets, based on their previous materialization (and if so, how to do it).

Context: I am materializing assets, that fetch data from some external datasource, which is quite time consuming. The data is some form of table, and each row has a value "updated_at". Since the external source allows for filtering the data to be fetched according to this updated_at field, I would like to re-materialize assets by 1) loading the data from the old materialization and then 2) only fetch new data from the external source. 

Is it possible to do this? Or would you see a better way of doing this (then the described procedure)?

All the best and thanks in advance,
Jakob

**Comment by jamiedemaria** [comment]
Hi! You can partition your assets and then depend on previous partitions of the asset

here's a related stack overflow question
https://stackoverflow.com/questions/75261880/how-do-i-write-a-dagster-asset-that-depends-on-an-earlier-partition-of-itself

and our general documentation on partitions https://docs.dagster.io/concepts/partitions-schedules-sensors/partitions
